site/docs/skylark/depsets.md

```diff
@@ -12,7 +12,8 @@ for authors of rules and aspects, but probably not macros.
 
 The main feature of depsets is that they support a time- and space-efficient
 merge operation, whose cost is independent of the size of the existing contents.
```

Comment:
Let's replace "merge" with "union" throughout, as that's the correct term for combining sets.

The grammar here isn't quite right. Let's change the whole paragraph to:
"""
The defining feature of depset is its time- and space-efficient union operation. The `depset` constructor accepts a list of elements ("direct") and a list of other sets ("transitive"), and returns a depset representing a set containing all the direct elements and the union of all the transitive sets. Conceptually, the constructor creates a new graph node that has the direct and transitive nodes as its successors. Depsets have a well-defined ordering semantics, based on traversal of this graph.
"""

Reply:
done